toolkits/global/packages/global-popup/js/index.js

```diff
@@ -0,0 +1,27 @@
+import {makeArray} from '../../global-javascript/src/helpers';
+import {Popup} from './popup';
+
+/**
+ * Data Attribute API
+ */
+
+const popup = () => {
+	const triggers = makeArray(document.querySelectorAll('[data-popup]'));
+	if (triggers.length === 0) {
+		return;
+	}
+	triggers.forEach(trigger => {
+		if (!trigger.hasAttribute('data-popup-target')) {
+			return;
+		}
+		const targetElementSelector = trigger.getAttribute('data-popup-target');
+		if (!targetElementSelector) {
```

Comment:
Is it worth doing a check for 
!document.querySelector(targetElementSelector)
as well